setup.py

```diff
@@ -23,12 +23,7 @@
 import pkg_resources
 import setuptools
 
-# The third party library path is defined in common.py,
-# which is required to import typing_extensions.
-from scripts import common # isort:skip pylint: disable=unused-import
```

Comment:
This is removed since we no longer use typing_extensions here, right?

Reply:
Yes.